### ISSUE TITLE
Non POT textures don't tile correctly on materials in WebGL 1

### ISSUE THREAD

**Issue by yaustar** [playcanvas/engine#3183]
Thread: https://forum.playcanvas.com/t/tiling-not-working-in-safari-macos/20299
Reproducible: https://playcanvas.com/editor/scene/1159541

![image](https://user-images.githubusercontent.com/16639049/119131376-4f357780-ba31-11eb-97a9-c69930bddf3b.png)

**Comment by willeastcott** [comment]
This is correct. Non-POT textures cannot be mipmapped or tiled/repeated under WebGL 1. This is what the WebGL 1 spec defines. WebGL 2 lifts these restrictions.

**Comment by yaustar** [comment]
Hmm, we should add a warning on the material inspector in that case if tiling is non 1
